e2e/smoke-tests/sample-apps/compat.js

```diff
@@ -82,7 +82,7 @@ async function authLogout() {
  *
  * Call a deployed function.
  * This cloud function must be deployed in this project first. See
- * e2e/README.md for more info.
+ * s/README.md for more info.
```

Comment:
`s/ReadME.md` doesn't seem right.

Reply:
Thanks, must have been trying to type shortcuts while in the editor,